app/models/document.rb

```diff
@@ -37,6 +37,7 @@ class Document < ApplicationRecord
   has_one_attached :upload
 
   before_save :set_display_name
+  before_create { client&.touch(:last_response_at) }
```

Comment:
Only on create because we are allowing volunteers to update attributes on the document. Currently, volunteers can't add documents but if we change to allowing them to add docs then we'll need to revise the logic.

Reply:
That makes sense.